Python/Product/VSInterpreters/PackageManager/PipPackageManager.cs

```diff
@@ -716,8 +716,10 @@ private sealed class Suppressed : IDisposable {
         }
 
         private void OnChanged(object sender, FileSystemEventArgs e) {
-            if ((Directory.Exists(e.FullPath) && !"__pycache__".Equals(PathUtils.GetFileOrDirectoryName(e.FullPath))) ||
-                ModulePath.IsPythonFile(e.FullPath, false, true, false)) {
+            if ((Directory.Exists(e.FullPath) && 
+                !PathUtils.GetFileOrDirectoryName(e.FullPath).Equals("__pycache__", StringComparison.InvariantCulture)) ||
+                ModulePath.IsPythonFile(e.FullPath, false, true, false)
+            ) {
```

Comment:
Searching for `.Equals(` in the code base shows several examples that compare file names with `StringComparison.OrdinalIgnoreCase`, and I think that's what we should use here.

Reply:
Will be modified in next commit